test/List.test.jsx

```diff
@@ -0,0 +1,53 @@
+import { fireEvent, render } from '@testing-library/react';
+
+import List from '../src/List';
+
+describe('List', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  let mockTasks = null;
+  const handleClick = jest.fn();
+
+  const renderComponent = (tasks = []) => {
+    const { container, getAllByRole, getByRole } = render(
+      <List
+        tasks={tasks}
+        onClickDelete={handleClick}
+      />,
+    );
```

Comment:
```suggestion
    const { container, getAllByRole, getByRole } = render((
      <List
        tasks={tasks}
        onClickDelete={handleClick}
      />
    ));
```

괄호를 두번 감싸주면 콤마를 없앨 수 있어요!

Reply:
넵넵!